docs/admins/truenas.md

```diff
@@ -0,0 +1,40 @@
+#  TrueNAS
+
+## General
+----
+We have a NAS unit running TrueNAS at http://vault.synshop.net (http://10.0.40.30). Any member who requires some storage space for project related storage can request an account to be created for them.
+
+- There is also a folder called "stash" that is a public, general temporary space that is mounted as a shared storage by all shop computers to quickly and easily move files between devices.
+
+- The Vault pool has a folder called "members" under which each user with an active account on the NAS has a personal folder.
+- The stash folder is at the same level.
+- Additional folders can be created as needed.
+
+- There are two user groups: Members and Admins. Folders can be given Members, Admin, or selected individual permissions as needed. More groups can be created if required but is not advised.
+
+## Administration
+----
+To add a new user to the system, you need to log into the web interface (http://vault.synshop.net), which is only possible with the root user. (Credentials are in the administrative keepass).
```

Comment:
> administrative keepass

Oh!  I would have expected it in the Systems keepass instead!  Are there other web/system logins in there?  We should decided which is best and consolidate.

also, distilling down some of the wording:

```suggestion
To add a new user to the system, you need to log into the [web interface](http://vault.synshop.net) as `root` user. Credentials for `root` in the administrative keepass in Google Drive under `TrueNAS`.
```